WebRoot/js/zimbraAdmin/resource/view/ZaNewResourceXWizard.js

```diff
@@ -572,7 +576,7 @@ ZaNewResourceXWizard.myXFormModifier = function(xFormObject) {
                 msgName:ZaMsg.NAD_zimbraPrefCalendarAutoAcceptSignatureId,
                 width: "280px",
                 label:ZaMsg.NAD_zimbraPrefCalendarAutoAcceptSignatureId, labelLocation:_LEFT_,
-                visibilityChecks:[],
+                visibilityChecks:[ZaItem.hasWritePermission],
```

Comment:
as mentioned earlier can we move this check to enableDisableChecks?

Reply:
The reason is the same as the above. A field should be hidden if we don't have write permission on New Resource wizard.